plugins/Installation/lang/en.json

```diff
@@ -1,14 +1,14 @@
 {
     "Installation": {
-        "CollaborativeProject": "Matomo is a collaborative project, built with love by people from all over the world.",
-        "ConfigurationHelp": "Your Matomo configuration file appears to be misconfigured. You can either remove config\/config.ini.php and resume installation, or correct the database connection settings.",
-        "ConfirmDeleteExistingTables": "Are you sure you want to delete the tables: %s from your database? WARNING: DATA FROM THESE TABLES CANNOT BE RECOVERED!",
+        "CollaborativeProject": "Matomo is a collaborative project built with love.",
+        "ConfigurationHelp": "Correct your Matomo configuration file by either either removing config\/config.ini.php and resuming installation, or correct the database connection settings.",
+        "ConfirmDeleteExistingTables": "Delete the %s tables from your database? WARNING: DATA FROM THESE TABLES CANNOT BE RECOVERED!",
```

Comment:
The existing strings are a bit clearer here, explicitly saying "Your Matomo configuration file appears to be misconfigured" and "Are you sure you want to delete...". Recommend reverting this change

Reply:
"Appears to" is unclear in my view.
"Are you sure" has fallen out of fashion.